source/_integrations/analytics.markdown

```diff
@@ -0,0 +1,158 @@
+  "huuid": "12a3456bc78d90123ef4567g789012h3",
+  "version": "2021.4.0",
+  "installation_type": "Home Assistant OS",
+  "supervisor": {
+    "healthy": true,
+    "supported": true
+  }
+}
+```
+
+{% enddetails %}
+
+## Usage analytics
+
+You need to enable [basic analytics](#basic-analytics) to be able to enable this.
+
+This includes:
+
+- The names of all your integrations
+
+If your system include the Supervisor this will also contain:
+
+- The name of all installed add-ons
+- The version of all installed add-ons
+- The state of protection mode of all installed add-ons
+- The state of the auto update toggle of all installed add-ons
+
+{% details Example payload %}
+
+```json
+{
+  "huuid": "12a3456bc78d90123ef4567g789012h3",
+  "version": "2021.4.0",
+  "installation_type": "Home Assistant OS",
+  "supervisor": {
+    "healthy": true,
+    "supported": true
+  },
+  "integrations": ["awesome_integration"],
+    "addons": [
+        {
+            "slug": "awesome_addon",
+            "protected": true,
+            "version": "1.0.0",
+            "auto_update": false
+        }
+    ]
+}
+```
+
+{% enddetails %}
+
+## Statistics
+
+You need to enable [basic analytics](#basic-analytics) to be able to enable this.
+
+This includes:
+
+- Number of integrations
+- Number of users
+- Number of entities
+- Number of automations
+
+If your system include the Supervisor this will also contain:
+
+- Number of installed add-ons
+
+{% details Example payload %}
+
+```json
+{
+  "huuid": "12a3456bc78d90123ef4567g789012h3",
+  "version": "2021.4.0",
+  "installation_type": "Home Assistant OS",
+  "supervisor": {
+    "healthy": true,
+    "supported": true
+  },
+  "state_count": 1,
+  "automation_count": 2,
+  "integration_count": 3,
+  "addon_count": 4,
+  "user_count": 5
+}
+```
+
+{% enddetails %}
+
+## Diagnostics
+
+This is not sent to the same [Receiver](#receiver) as the rest of the information enabled with this integration. Crash reports are sent to [Sentry](https://sentry.io/welcome/). This data will not be made public and will only be available to a few core developers.
+
+For now, this is only being done for events inside the Supervisor if your system has that. If this changes to include other parts of the ecosystem it will be mentioned in the release notes and the documentation here will be updated.
+
+## Receiver
+
+The receiver that the payloads are sent to is running as a [CloudFlare Worker](https://workers.cloudflare.com/) and the code for that can be inspected in the [home-assistant/analytics repository](https://github.com/home-assistant/analytics.home-assistant.io).
+
+## Storage and usage
+
+When your installation send a payload, that payload includes a unique identifier, this identifier is used to make sure that future payload updates, only changes information from your installation.
+The data is stored in CloudFlare's KV (Key-Value) store for the [Receiver](#receiver), and will be stored for a maximum of 60 days since last update.
```

Comment:
```suggestion
When your installation sends a payload, that payload includes a unique identifier, this identifier is used to make sure that future payload updates, only change information from your installation.
The data is stored in CloudFlare's KV (Key-Value) store for the [Receiver](#receiver), and will be stored for a maximum of 60 days since the last update.
```